charts/housnav/values-uat.yaml

```diff
@@ -10,11 +10,11 @@ readinessProbe:
 
 resources:
   requests:
-    memory: "50Mi"
-    cpu: "25m"
-  limits:
     memory: "100Mi"
     cpu: "50m"
+  limits:
```

Comment:
What inspired the change here?

Reply:
> What inspired the change here?

I did a more accurate load test with Apache JMeter and found the optimal value for the moment. These values take into account the limits we have on this project.